webapp/src/store/comments.ts

```diff
@@ -9,16 +9,29 @@ import {loadBoardData, initialReadOnlyLoad} from './initialLoad'
 
 import {RootState} from './index'
 
+type CommentsState = {
+    comments: {[key: string]: CommentBlock}
+    commentsByCard: {[key: string]: CommentBlock[]}
```

Comment:
Adding the list of comments by card to simplify querying this and avoid re-renders on unneeded updates.